### PR TITLE
Setup a search service based on elasticsearch

### PR DESCRIPTION
**Pull request by webeautiful**

### CODE REVIEW

File: search/jdbc.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+zipfile=elasticsearch-jdbc-2.3.1.0-dist.zip
+river=jdbc
+if [ ! -f $zipfile ];then
+    wget http://xbib.org/repository/org/xbib/elasticsearch/importer/elasticsearch-jdbc/2.3.1.0/$zipfile
+fi
+if [ ! -d $river ];then
+    unzip elasticsearch-jdbc-2.3.1.0-dist.zip
+    mv elasticsearch-jdbc-2.3.1.0 $river
+fi
+bin=$river/bin
```
Comment (liushuigs) [inline search/jdbc.sh]: It's better to spit this file to two, one is for setting up, and the other is for creating index

File: search/jdbc.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+lib=$river/lib
+echo '{
+    "type" : "jdbc",
+    "jdbc" : {
+        "url" : "jdbc:mysql://localhost:3306/cavat",
+        "user" : "common",
```
Comment (liushuigs) [inline search/jdbc.sh]: It will be better to move config to a separate config file

Comment (webeautiful) [inline search/jdbc.sh]: I think so, this is only a demo. I am planning to write a script file specially to import elasticsearch from mysql.

File: search/jdbc.sh (hunk 1 above)
Comment (webeautiful) [inline search/jdbc.sh]: I think so, this is only a demo. I am planning to write a script file specially to import elasticsearch from mysql.

File: search/jdbc.sh (hunk 2 above)
Comment (webeautiful) [inline search/jdbc.sh]: Ok. Let me think what to do

**Comment by liushuigs** [comment]
LGTM

**Pull request merged**